go/database/log/schema/collection.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE collection (
+                        id text PRIMARY KEY,
+                        record_compaction_offset_position bigint NOT NULL,
+                        record_enumeration_offset_position bigint NOT NULL
```

Comment:
What is the intent of record_enumeration_offset_position?

Reply:
I am writing a notion around it, but it is to handle the case when we purge log, we need to know what was the last offset for the collection.